scripts/route_check.py

```diff
@@ -687,85 +692,102 @@ def check_routes():
     :return (0, None) on sucess, else (-1, results) where results holds
     the unjustifiable entries.
     """
-    intf_appl_miss = []
-    rt_appl_miss = []
-    rt_asic_miss = []
-    rt_frr_miss = []
+    namespace_list = []
+    if namespace is not multi_asic.DEFAULT_NAMESPACE and namespace in multi_asic.get_namespace_list():
+        namespace_list.append(namespace)
+    else:
+        namespace_list = multi_asic.get_namespace_list()
```

Comment:
In MULTI-ASIC devices,   can you check if get_namespace_list() returns DEFAULT_NAMESPACE which is ''. if so we can skip that